src/app/migrations/index.js

```diff
@@ -65,10 +66,13 @@ module.exports = function(done) {
       '1.21.0-dev.0': require('./1.21.0')
     };
     const migrate = require('app-migrations')(migrations);
+
     migrate(previousVersion, currentVersion, function(err2, res) {
       if (err2) {
-        return debug('error', err2);
+        debug('error', err2);
+        return done(err2);
```

Comment:
This change we should definitely keep as it causes the spinning instead of giving a proper error - because it would try to "downgrade" and see there were migrations...

Reply:
Yeah i think this is good in general, but the migration when you downgrade won't necessarily fail or? I mean migration interfaces usually have an `up` and `down` method but it feels we are only go up.

Reply:
No check the migration scripts: There is no `down` https://github.com/mongodb-js/compass/blob/fix-migrations-hanging-on-placeholder-version/src/app/migrations/1.21.0.js - or am I totally missing it? :D